### PR TITLE
creating a file for good pizza in Cleveland

### LINKED ISSUE #8: People have issues...from christiance
# Overview

Hey Markdowners!
- here's a point
- here's another point
- here's **an important** point
- [ ] use a checkbox
- [ ] add in a second
- [ ] finish up!

### PR DESCRIPTION
**Pull request by christianCE**
This is a MD text file to keep a list of the best places to grab a slice where it rocks!
- this also probably takes MD
- just checking

Closes #8.

### CODE REVIEW

File: Cleveland Pizza.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+##Notes for Cleveland Rocks!
+
+
+#good places for Pizza:
```
Comment (brianamarie) [inline Cleveland Pizza.md:4]: This is making me hungry for 🍕 !!!

**Comment by aparentMMS** [comment]
hello cleveland

File: Cleveland Pizza.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+* Crostata's (East Side)
```
Comment (apoorva89) [inline Cleveland Pizza.md:5]: That sounds good like a good :pizza: place

File: Cleveland Pizza.md (hunk 1 above)
Comment (christianCE) [inline Cleveland Pizza.md:4]: me too! Had to settle for tuna salad :disappointed:

**Comment by apoorva89** [comment]
@christianCE looks great :ship:

File: Cleveland Pizza.md (hunk 2 above)
Comment (christianCE) [inline Cleveland Pizza.md:5]: yup! best in Cleveland, they say...

http://www.crostatas.com/

**Pull request merged**